ide/winvs/user_settings.h

```diff
@@ -11,7 +11,7 @@
 #define HAVE_ECC
 #define HAVE_AESGCM
 #define HAVE_HASHDRBG
-#define WOLFSSL_AESCTR
+#define WOLFSSL_AES_COUNTER
```

Comment:
Oh yeah, the flag is `WOLFSSL_AES_COUNTER`. I'm sorry for the wrong request.

Reply:
But still not seem to work. Don't see why.